packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -185,7 +185,7 @@ export interface IContainerRuntimeBase extends IEventProvider<IContainerRuntimeB
 	 * @param content - Content of the signal.
 	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	submitSignal(type: string, content: any, targetClientId?: string): void;
+	submitSignal(type: string, content: unknown, targetClientId?: string): void;
```

Comment:
We should use the `submit: (...) => ...` syntax for interfaces as we did in #20251 to help ensure implementations match requirements.
And for line 458.